.gitignore

```diff
@@ -7,7 +7,6 @@
 .*.sw*
 .*.un~
 clover.xml
-composer.lock
```

Comment:
Only when $DEPS is either 'latest' or 'lowest' the ```composer.lock``` is overwritten. When $DEPS is either 'locked' or not set at all the original ```composer.lock``` will be used. So no problem from my POV. Or am I missing something completely out?